modules/vaos/app/services/concerns/token_authentication.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+# TokenAuthentication provides common functionality for services that require
+# bearer token authentication. It handles token retrieval, caching, and header generation.
+module TokenAuthentication
```

Comment:
Note: This is a refactor of the token code from the base service classes. There were no code changes here.